### PR TITLE
Fix waiting for node upgrades

### PR DESCRIPTION
**Pull request by wojtek-t**
Fix https://github.com/kubernetes/kubernetes/issues/27852

If someone wants to finish it, feel free to pick it up and finish.

@davidopp

### CODE REVIEW

**Comment by zmerlynn** [comment]
Not it. Not today. :)

**Comment by wojtek-t** [comment]
@mikedanese - feel free to ignore it today. It's not yet finished.

File: cluster/gce/upgrade.sh [hunk 1]
```diff
@@ -230,10 +236,34 @@ function do-node-upgrade() {
...
+    result=$(gcloud alpha compute rolling-updates \
+        --project="${PROJECT}" \
+        --zone="${ZONE}" \
+        describe \
+        ${update} \
+        --format='value(status)' || true)
+    if [ $result != "ROLLED_OUT" ]; then
+      sleep 10
+      echo ${result}
+		fi
```
Comment (mikedanese) [inline cluster/gce/upgrade.sh]: nit: tabs to spaces

File: cluster/gce/upgrade.sh [hunk 2]
```diff
@@ -208,7 +208,6 @@ function prepare-node-upgrade() {
   fi
   create-node-instance-template "${template_name}"
   # The following is echo'd so that callers can get the template name.
```
Comment (piosz) [inline cluster/gce/upgrade.sh]: Remove the comment also.

Comment (wojtek-t) [inline cluster/gce/upgrade.sh]: done

**Comment by piosz** [comment]
LGTM

**Comment by wojtek-t** [comment]
@davidopp - FYI

**Pull request merged**